storage/innobase/os/os0file.cc

```diff
@@ -4140,6 +4157,8 @@ os_aio_array_free(
 	if (srv_use_native_aio) {
 		ut_free(array->aio_events);
 		ut_free(array->aio_ctx);
+		ut_free(array->pending);
+		ut_free(array->count);
```

Comment:
Debug-build assert that arrays (all of them; not just the read one) are empty on shutdown.

Reply:
Done